brainstormforce/ultimate-addons-for-gutenberg
# Fix: Responsive condition - Removed extra toggle added and resolve bottom divider missing issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/999e91fd-4a41-4ad0-ab8e-0e33a57b6554?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Removed extra toggle added fixed - https://share.getcloudapp.com/d5u9PYrm
- Divider missing issue fixed - https://share.getcloudapp.com/p9u6rlpy

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop core and Spectra block.
- Add a responsive condition to check it is working as expected or not.
- Refresh the editor setting is saved or not after updating the page.
- Check divider is visible when tab is open and closed for both core/spectra blocks.

### Checklist:
